lib/pluggy_elixir/config.ex

```diff
@@ -0,0 +1,83 @@
+defmodule PluggyElixir.Config do
+  @moduledoc """
+  PluggyElixir client configurations
+
+  To use `PluggyElixir` you must add into your config file `config.exs` the
+  required config:
+
+      config :pluggy_elixir,
+        client_id: "your-app-client-id",
+        client_secret: "your-app-client-secret",
+
+
+  Allowed configuration:
+
+  - client_id(required): A client id provide by Pluggy
+  - client_secret(required): A client secret provide by Pluggy
+  - non_expiring_api_key: A boolean value to enable create a non expiring API
+  KEY. (default is `false`)
+  - sandbox: A boolean value to enable using sandbox `Connector`. (default is `false`)
+  - host: A string containing the API host. (default is `"api.pluggy.ai"`)
+
+  """
+
+  @default_host "api.pluggy.ai"
+
+  @doc false
+  def get_client_id, do: get_pluggy_elixir_config(:client_id)
+
+  @doc false
+  def get_client_secret, do: get_pluggy_elixir_config(:client_secret)
+
+  @doc false
+  def get_host_uri do
+    ~r"(?<scheme>http[s]?)?(:[\/]{2})?(?<host>[\w\.]+)(:?(?<port>\d+))?(\/(?<path>.*))?"
+    |> Regex.named_captures(get_pluggy_elixir_config(:host, @default_host))
+    |> to_uri()
+  end
+
+  @doc false
+  def non_expiring_api_key,
+    do: if(get_pluggy_elixir_config(:non_expiring_api_key, false) == true, do: true, else: false)
```

Comment:
WDYT
``` 
def non_expiring_api_key, do: get_config(:non_expiring_api_key, false)
```

Reply:
This `if` is to avoid non boolean values.

Reply:
I agree with @ecamalionte, it seems to me that this protection could be removed, assuming that this contract/type will be respected.

Reply:
The problem is that. We can not ensure the contract/type will be respected. The better way would be validate all config and return a error message for invalid configuration. But its to early to add that kind of protection. The low cost is add this if and avoid security issues.